pylint/checkers/base.py

```diff
@@ -402,8 +402,7 @@ class BasicErrorChecker(_BasicChecker):
                   'return-arg-in-generator',
                   'Used when a "return" statement with an argument is found '
                   'outside in a generator function or method (e.g. with some '
-                  '"yield" statements).',
```

Comment:
This check should actually be completely removed, otherwise it will fail when pylint is executed with Python 3.